src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php

```diff
@@ -1164,6 +1180,20 @@ public function iShouldBeInformedThatMyPromotionIsNoLongerApplied(PromotionInter
         );
     }
 
+    /**
+     * @Then I should not be able to confirm order because the :shippingMethod shipping method is not available
+     */
+    public function iShouldNotBeAbleToConfirmOrderBecauseTheShippingMethodIsNotAvailable(ShippingMethodInterface $shippingMethod): void
```

Comment:
There is no need to use a transformer for getting the entire entity as the name will be enough 🖖🏻 
```suggestion
     * @Then I should not be able to confirm order because the :shippingMethodName shipping method is not available
     */
    public function iShouldNotBeAbleToConfirmOrderBecauseTheShippingMethodIsNotAvailable(string $shippingMethodName): void
```